rapid7_insightvm/komand_rapid7_insightvm/actions/get_asset_vulnerabilities/action.py

```diff
@@ -3,8 +3,7 @@
 # Custom imports below
 import asyncio
 from komand_rapid7_insightvm.util import endpoints
-from komand_rapid7_insightvm.util.resource_helper import ResourceHelper
-from operator import itemgetter
```

Comment:
Removing this import causes an unresolved reference to the function on lines 80, 81, 85